### ISSUE TITLE
how to train with pre-trained model?

### ISSUE THREAD

**Issue by moonuke** [huanghoujing/person-reid-triplet-loss-baseline#14]
have you pre-train on imagenet?
i wanna to train other cnn-net on your baseline and i didn't find pre-train script.

**Comment by huanghoujing** [comment]
I use ResNet-50 which is pretrained on ImageNet by pyTorch team. So I just load the officially provided ImageNet weight and then train the ReID model. For your cnn-net, you can find whether there is public pretrained weight. Otherwise, you have to train it on ImageNet as you want the ImageNet weight.
